pkg/gci/gci.go

```diff
@@ -90,6 +91,10 @@ func ProcessFiles(fileGenerator io.FileGeneratorFunc, cfg config.Config, fileFun
 		return err
 	}
 	for _, file := range files {
+		if cfg.SkipVendor && isInsideVendorDir(file.Path()) {
```

Comment:
Could you modify `GoFilesInPathsGenerator` to skip vendor directly? If so, we will reduce check times.